src/pages/VenueLandingPage/VenueLandingPageContent.tsx

```diff
@@ -46,6 +47,8 @@ const VenueLandingPageContent: React.FC<VenueLandingPageContentProps> = ({
 
   const venueId = useVenueId();
 
+  const { world } = useCurrentWorld({ worldId: venue?.worldId });
```

Comment:
[NOTE]
Now there is `useWorldById()` which should probably be used in combination/parallel with `useWorldById()` instead of older hooks.

Reply:
Changed in https://github.com/sparkletown/sparkle/pull/2546/commits/08a3290e454507ee3163e87d8f4078506f8d7b8f. 
This new hook looks better